docs(stripe): adding docs for stripe (#863)
front-commerce/developers.front-commerce.com: docs/03-extensions/stripe/reference/environment-variables.mdx

---
title: "Environment variables"
description:

>   "This reference lists environment variables used by the Stripe extension"

Reviewer: ```suggestion
  "This reference lists environment variables used by the Stripe extension."
```

Author: fixed

sidebar_position: 1
---

<p>{frontMatter.description}</p>

:::info

Your API keys can be found on your
[Stripe Dashboard](https://dashboard.stripe.com/account/apikeys)

:::

| Name                                    | Description                                                                                                                                     | Default value |
| --------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |

> | FRONT_COMMERCE_STRIPE_SECRET_KEY        | **Required** Your Stripe Secret key                                                                                                             | `undefined`   |
> | FRONT_COMMERCE_STRIPE_PUBLISHABLE_KEY   | **Required** Your Stripe Secret key                                                                                                             | `undefined`   |

Reviewer: **thought:** if a variable is required, there is no default value.

**suggestion:** remove them.

```suggestion
| FRONT_COMMERCE_STRIPE_SECRET_KEY        | **Required** Your Stripe Secret key                                                                                                             |    |
| FRONT_COMMERCE_STRIPE_PUBLISHABLE_KEY   | **Required** Your Stripe Secret key                                                                                                             |    |
```

Author: removed

> | FRONT_COMMERCE_STRIPE_MAX_NETWORK_RETRY | Max network retry to Stripe network before consider call as a failure                                                                           | `3`           |

Reviewer: **thought:** [In the v2 documentation](https://developers.front-commerce.com/docs/2.x/advanced/payments/stripe#advanced-network-retries), this environment variable deserved a whole section.

**question:** Is there a reason that isn't the case anymore?

Author: There's no acceptance criterias in the inital issue and i think it speak by itself. I've added a link to the stripe documentation.

> | FRONT_COMMERCE_STRIPE_MAX_NETWORK_RETRY | Max network retry to Stripe network before consider call as a failure                                                                           | `3`           |

Reviewer: ```suggestion
| FRONT_COMMERCE_STRIPE_MAX_NETWORK_RETRY | The maximum number of network tries should be made to the Stripe network before the call is considered a failure.  | `3`           |
```

Author: fixed
